pkg/network/render.go

```diff
@@ -16,17 +16,15 @@ import (
 
 func Render(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.BootstrapResult, manifestDir string) ([]*uns.Unstructured, error) {
 	log.Printf("Starting render phase")
-	objs := []*uns.Unstructured{}
 
 	// render Multus
-	o, err := RenderMultus(conf, manifestDir)
+	objs, err := RenderMultus(conf, manifestDir)
 	if err != nil {
 		return nil, err
 	}
-	objs = append(objs, o...)
```

Comment:
The original version of this has a bunch of pieces that all look exactly the same, but now this rewrite makes the first clause look different. I think if you changed the initial `objs := []*uns.Unstructured{}` to `var objs []*uns.Unstructured` then that would get rid of the warning without needing to rewrite this part